src/UI/Art/SpriteArtist/PlayerSetupCommanderArtist.java

```diff
@@ -565,7 +567,7 @@ public static void initBins(final int selectedCO)
       }
       OuterCategoryPanel outerCategory = new OuterCategoryPanel(outerPalette, outerName);
 
-      for( int innerIndex = 0; innerIndex < allCmdrLists.length; ++innerIndex )
+      for( int innerIndex = 0; innerIndex < innerMax; ++innerIndex )
```

Comment:
Curious: This looks like the only thing that is actually different after the `innerMax` change, but I don't see a difference in behavior in the menu. What was the actual problem here?

Reply:
Yessir. It was using the number of outer categories as the bound for the inner category's length.
This resulted in not finding some COs in the collection, so they wouldn't show up in the GUI (but did sometimes affect control, which was weird).
The CO(s) that didn't show up was semi-random per run due to hash ordering, which was pretty weird to diagnose.

Reply:
Huh. Guess I've just gotten lucky.